.github/workflows/acceptance-tests.yml

```
@@ -117,7 +117,15 @@ jobs:
       fail-fast: false
       max-parallel: 1
       matrix:
-        tests: [publicProject, updateProjects, useSession]
+        tests:
+          [
+            publicProject,
+            privateProject,
+            updateProjects,
+            testDatasets,
+            useSession,
+            checkWorkflows,
+          ]
```

Use the nicer list format for yaml:

```suggestion
        tests:
          - publicProject
          - privateProject
          - updateProjects
          - testDatasets
          - useSession
          - checkWorkflows
```